<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcdetailTime</class>
 <widget class="QWidget" name="calcdetailTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculation Item Time</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&lt;h1&gt;Calculation Part 'Time'&lt;/h1&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Calculate time efforts here for one unit of the template. &lt;br/&gt;Note that the costs may depend on a global hourly rate.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel3">
       <property name="text">
        <string>&amp;Label:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>m_nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_nameEdit">
       <property name="text">
        <string>Work</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel3_2">
       <property name="text">
        <string>&amp;Time effort:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>m_dauer</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QSpinBox" name="m_dauer">
         <property name="value">
          <number>5</number>
         </property>
         <property name="minValue" stdset="0">
          <number>0</number>
         </property>
         <property name="maxValue" stdset="0">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="_cbTimeUnit"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel3_2_2">
       <property name="text">
        <string>&amp;Hourly rate:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>m_hourSets</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="m_hourSets"/>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="m_stdGlobal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Apply the &amp;global hourly rate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_nameEdit</tabstop>
  <tabstop>m_dauer</tabstop>
  <tabstop>m_hourSets</tabstop>
  <tabstop>m_stdGlobal</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
